web/src/settings_data.ts

```diff
@@ -1,3 +1,5 @@
+import $ from "jquery";
```

Comment:
This module is intentionally a data module that does't have any DOM side effects, so we shouldn't import jQuery here.

Reply:
Fixed.